build.gradle.kts

```diff
@@ -49,20 +53,41 @@ tasks {
     }
 }
 
+// override release versions based on the current branch
+apply(from ="./gradle-scripts/gitBranch.gradle.kts")
```

Comment:
is there a more elegant way to add a suffix to the version name?

Reply:
Depends on what you find lacking in elegance? This seems ported from kaluga so it uses some kaluga constructs (usings extras etc). should not be hard to port this to something like `versionSuffix = someMethod()` etc. e.g. using `buildSrc` if that is what you mean.

Reply:
since we use this across platform the most elegant way would probably to turn it into a plugin.